modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -193,6 +194,9 @@ For more information about managing a large number of clients, see xref:speciali
 | Minimum 150 GB
 
 |===
+//        ^^^
+// CHECK  ^^^ or shall we change this back to 100 GB?
```

Comment:
I don't know from where the 150GB comes from, so we'll need to check with the original requester for it for the reasoning.

From my POV, 100GB is quite enough.

Reply:
I think it happened with my https://github.com/uyuni-project/uyuni-docs/pull/3249 --I quickly checked the chain-linked issue and reports. I myself probably invented the 150 GB value.  I'll change it back to 100 GB.